src/System.Text.Json/tests/RealWorld/BlogPost.cs

```diff
@@ -0,0 +1,850 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Text.Json.Serialization;
+using System.Text.RegularExpressions;
+using Xunit;
+
+namespace System.Text.Json.Tests.RealWorld.BlogPost
+{
+    public class Test : RealWorldTest
```

Comment:
We're only ever going to have a single test based on incorporating content from a "BlogPost"?